server-side/email-service/app/mailer.js

```diff
@@ -0,0 +1,50 @@
+const nodemailer = require('nodemailer');
+const Bottleneck = require('bottleneck');
+const retry = require('async-retry');
+
+let transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'prwyyqtsly@gmail.com',
```

Comment:
save email & password in .env
(Is this the address who sends the emails? please change to understandable address, if possible)